_posts/2016-09-07-kubelet.markdown

```diff
@@ -0,0 +1,508 @@
+---
+layout:     post
+title:      "kubelet源码分析"
+subtitle:   " \"kubelet source code notes\""
+date:       2016-09-07 00:00:00
+author:     "zhangweiwei"
+header-img: "img/2016-09-07-kubelet/header.jpg"
+catalog: true
+tags:
+    - etcd
```

Comment:
tag should be `kubernetes` or `kubelet` or something else which can represent the content of the blog.